whatlies/common.py

```diff
@@ -24,7 +32,32 @@ def handle_2d_plot(embedding, kind, color=None, xlabel=None, ylabel=None, show_o
         plt.quiver([0], [0], [embedding.vector[0]], [embedding.vector[1]], color=color,
                    angles='xy', scale_units='xy', scale=1)
         plt.text(embedding.vector[0] + 0.01, embedding.vector[1], name)
-    if kind == "text":
+    if (kind == "text") or annot:
```

Comment:
i like this touch! but i'll likely change the term to `annotate` later. 

value(clarity) > value(saving some characters)

Reply:
You don't have to change this for the PR if you don't feel like it. It is a minor details for this phase of the project.

Reply:
I guess I was inspired by the seaborn heatmap (I've used that a lot lately), they use annot :)